src/pages/ConnectionCompletePage.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+
+
+function ConnectionCompletePage() {
+    const theme = useTheme();
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    return (
+        <View style={styles.deeplinkWrapperContainer}>
+            <View style={styles.deeplinkWrapperMessage}>
+                <View style={styles.mb2}>
+                    <Icon
+                        width={variables.modalTopIconWidth}
```

Comment:
Can you make sure we show this illustration at 272 x 188?

Reply:
Those sizes don't match any of the sizes we have in [variables](https://github.com/Expensify/App/blob/18694a48f1693dd9a3f0d45780432c7c2b2b5a60/src/styles/variables.ts#L138), I'll just hardcode here I guess

Reply:
Works for me. Our illustrations are not all the same size, so I think that's okay.

Reply:
Setting height:188 and width:272 it looks like this now: 
<img width="1711" alt="image" src="https://github.com/Expensify/App/assets/87341702/b4d71ca4-8283-4559-a3f9-436bc50a0d26">

I think the space between the icon and the text got smaller because the height I had before was too much and was just filled with empty space

Reply:
Updated

Reply:
Exactly! So yeah, let's leave it like this then.